blocks/color-palette/index.js

```diff
@@ -1,88 +1,11 @@
-/**
- * External dependencies
- */
-import classnames from 'classnames';
-import { ChromePicker } from 'react-color';
-import { map } from 'lodash';
-
 /**
  * WordPress dependencies
  */
-import { Dropdown, withContext } from '@wordpress/components';
-import { __, sprintf } from '@wordpress/i18n';
+import { ColorPalette } from '@wordpress/components';
```

Comment:
Do we need to keep this file since we always use `PanelColor` in all core blocks?

Reply:
Yes, in order to be back compatible with blocks created by the community and tht use @blocks/ColorPalette.